view/library/send-to-editor.php

```diff
@@ -207,7 +213,9 @@ function kaltura_updateRatio() {
 				entryId    : <?php echo wp_json_encode($this->entry->id); ?>,
 				playersList: '#uiConfId',
 				dimensions : 'input[name=playerRatio]',
-				submit     : 'input[name=sendToEditorButton]'
+				submit     : 'input[name=sendToEditorButton]',
+				entryError: '<?php echo $this->entryError; ?>',
```

Comment:
Use wp_json_encode for `wp_json_encode` and `entryConverting`
It is required for proper escaping based on WP VIP rules and it will keep the correct variable type in JavaScript.